crates/turborepo-lib/src/commands/link.rs

```diff
@@ -667,7 +673,10 @@ mod test {
     async fn test_link_spaces() {
         // user config
         let user_config_file = NamedTempFile::new().unwrap();
-        fs::write(user_config_file.path(), r#"{ "token": "hello" }"#).unwrap();
+
+        // auth file
+        let auth_file = NamedTempFile::new().unwrap();
+        fs::write(auth_file.path(), r#"{ "token": "hello" }"#).unwrap();
```

Comment:
Isn't this the old token format getting written to the auth file path?

Reply:
you are correct, missed this